muikku-atests/src/test/java/fi/muikku/mock/PyramusMock.java

```diff
@@ -0,0 +1,634 @@
+package fi.muikku.mock;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlMatching;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import org.joda.time.DateTime;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.joda.JodaModule;
+
+import fi.muikku.TestUtilities;
+import fi.muikku.mock.model.MockCourseStudent;
+import fi.muikku.mock.model.MockLoggable;
+import fi.muikku.mock.model.MockStaffMember;
+import fi.muikku.mock.model.MockStudent;
+import fi.pyramus.rest.model.ContactType;
+import fi.pyramus.rest.model.CourseStaffMember;
+import fi.pyramus.rest.model.CourseStaffMemberRole;
+import fi.pyramus.rest.model.CourseStudent;
+import fi.pyramus.rest.model.CourseType;
+import fi.pyramus.rest.model.EducationType;
+import fi.pyramus.rest.model.EducationalTimeUnit;
+import fi.pyramus.rest.model.Email;
+import fi.pyramus.rest.model.Grade;
+import fi.pyramus.rest.model.GradingScale;
+import fi.pyramus.rest.model.Person;
+import fi.pyramus.rest.model.StaffMember;
+import fi.pyramus.rest.model.Student;
+import fi.pyramus.rest.model.StudyProgramme;
+import fi.pyramus.rest.model.StudyProgrammeCategory;
+import fi.pyramus.rest.model.Subject;
+import fi.pyramus.rest.model.WhoAmI;
+import fi.pyramus.webhooks.WebhookCourseStaffMemberCreatePayload;
+import fi.pyramus.webhooks.WebhookCourseStudentCreatePayload;
+import fi.pyramus.webhooks.WebhookPersonCreatePayload;
+import fi.pyramus.webhooks.WebhookStaffMemberCreatePayload;
+import fi.pyramus.webhooks.WebhookStudentCreatePayload;
+
+public class PyramusMock {
+  
+  private List<MockStudent> students = new ArrayList<>();
```

Comment:
by convention, private variables are usually introduced as last of the class (Injected properties make an exception to this but these are not injected properties)